compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/model/SymbolTable.java

```diff
@@ -185,6 +185,7 @@
 
     private Names names;
     public Map<BPackageSymbol, SymbolEnv> pkgEnvMap = new HashMap<>();
+    public Map<Name, BPackageSymbol> predeclaredModules = new HashMap<>();
```

Comment:
A minor thing. Was wondering whether we should make this an immutable map.

Reply:
Changed to immutable map.